doc/ref_model/chapters/chapter04.md

```diff
@@ -263,63 +263,178 @@ Table 4-10 shows performance measurement capabilities.
+where the specifications enclosed within "[" and "]" are optional, and the 'extra profile specs' are needed to capture special node configurations not accounted for by the profile and profile extensions.
 
-NOTE: On a Kubernetes based environment, resource allocation will directly depend on how resource requests are configured (please check [RA2](https://cntt.readthedocs.io/en/latest/ref_arch/kubernetes/) for additional details).
+Examples, node configurations specified as: B, B.low-latency, H,  and H.very-high-speed-network.very-low-latency-edge.
 
-<a name="Table4-13"></a>
+A workload needs to specify the configuration and capabilities of the infratsructure that it can un on, the size of the resources it needs, and additional information (extra-specs) such as whether the workload can share core siblings (SMT thread) or not, whether it has affinity (viz., needs to be placed on the same infrastructure node) with other workloads, etc. The capabilities required by the workload can, thus, be specified as:
 
+>  \<profile name>[.\<profile_extension>][.\<extra profile specs>].\<workload flavour specs>[.\<extra-specs>]
 
+where the \<workload flavour specs> are specified as defined in [4.2.4.3 Workload Flavours and Other Capabilities Specifications Format](#4.2.4.3) below.
 
+<a name="4.2.1"></a>
+### 4.2.1 Profiles
+
+<a name="4.2.1.1"></a>
+#### 4.2.1.1 Basic Profile
+
+Hardware resources configured as per the Basic profile (B) such that they are only suited for workloads that tolerate variable performance, including latency, and resource over-subscription. Only Simultaneous Multi-Threading (SMT) is configured on nodes supporting the Basic profile. With no NUMA alignment, the vCPUs executing processes may not be on the same NUMA node as the memory used by these processes. When the vCPU and memory are on different NUMA nodes, memory accesses are not local to the vCPU node and thus add latency to memory accesses. The Basic profile supports over subscription (using CPU Allocation Ratio) which is specified as part of sizing information in the workload profiles.
+
+<a name="4.2.1.2"></a>
+#### 4.2.1.2 High Performance Profile
+
+The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
 
 <a name="4.2.2"></a>
-### 4.2.2 Virtual Network Interface Specifications
+### 4.2.2 Profiles Specifications & Capability Mapping
+
+| Ref | Capability  | Basic | High Performance | Notes |
+|-----|---------|----------|----------|--------|
+| e.cap.006 | CPU pinning | No | Yes | Exposed performance capabilities as per Table 4-2. |
```

Comment:
@pgoyal01 CPU allocation ratio, i.cap.016, is missing

Reply:
@karinesevilla because that is part of flavour specification and not profile -- please see [4.2.4.2 Workloads Extra Capabilities Specifications](https://github.com/cntt-n/CNTT/blob/pgoyal01-patch-RM-42-new/doc/ref_model/chapters/chapter04.md#4242-workloads-extra-capabilities-specifications)

Reply:
@pgoyal01 As CPU allocation ratio needs to be configured consequently on the node, it can be also a profile capability

Reply:
@karinesevilla pleasee review -- made changes as per your email suggestion